dwellingly/src/App.js

```diff
@@ -81,14 +83,27 @@ export class App extends React.Component {
       <UserContext.Provider value={{ user: { ...this.state.userSession }, login: this.login, logout: this.logout }} >
         <BrowserRouter>
           <div className='App'>
+            <NavMenu />
             <Header />
-
             <Switch>
               <PrivateRoute exact path='/' component={Dashboard} />
               <Route exact path='/login' component={LoginForm} />
               <PrivateRoute exact path='/dashboard' component={Dashboard} />
               <Route exact path='/terms' component={Terms} />
-            </Switch>
+
+              <Route exact path='/home' />
```

Comment:
Since all these routes will require someone to be logged in, we'll have to use "PrivateRoute" instead of just "Route" like the dashboard above.